emukit/test_functions/quadrature_functions.py

```diff
@@ -31,15 +31,18 @@ def _hennig1D(x: np.ndarray) -> np.ndarray:
 
 
 def circular_gaussian(mean: np.float = 0., variance: np.float = 1.) -> Tuple[UserFunctionWrapper, Callable]:
-    """ 2D toy integrand that is a Gaussian on a circle
     """
-    return UserFunctionWrapper(_circular_gaussian), lambda x: _circular_gaussian(x, mean, variance)
+    2D toy integrand that is a Gaussian on a circle
+    """
+    func = lambda x: _circular_gaussian(x, mean, variance)
+    return UserFunctionWrapper(func), func
```

Comment:
`UserFunctionWrapper` has `.f` field, so returning `func` is probably redundant

Reply:
I did this as a kind of curtesy to users. For instance in case they want to plot the test function, they do not need to figure out how to call it though the wrapped version (might be a bit confusing). 

Our test functions at the moment seem to have various return structures. Some return the wrapped function, some the unwrapped function, and some return the parameter space in addition. Do you think it would be beneficial to have a common structure for all of them?

Reply:
Agreed. Let's open an issue about it to host the discussion

Reply:
OK, so I'll leave it like that for now since changing might involve changing notebooks etc.